docs/libraries.md

```diff
@@ -1,24 +1,26 @@
-# Libraries 
+# Libraries
 
 ## Overview
 This document is an overview of Armada client libraries.
 
 ## Python
-Here, we give an overview of the Armada Python client. 
+Here, we give an overview of the Armada Python client.
 
 ### Installation
 Prerequisites for installation include a working Python installation and poetry. You may wish to install the client into a virutalenv.
 
 Our python client bindings can be installed directly from the Armada repository.
  1. Checkout the [armada git repository](https://github.com/g-research/armada)
  2. From the root of the repository, run `make python` to generated needed gRPC code.
- 3. Install the client with poetry: `cd client/python && poetry install -v` 
+ 3. Install the client with poetry: `cd client/python && poetry install -v`
 
 ### Usage
 Once installed, the client can be imported with `import armada_client.client`.
 
 ### API Documentation
-For full documentation of our python module API, please see our [autogenerated python API docs](/python_armada_client)
+For full documentation of our python module API, please see our [autogenerated python API docs](./python_armada_client.md)
```

Comment:
This link is going to be broken once generated. If you access `https://armadaproject.io/api.md` -- or really any md file directly -- it gets downloaded.

This needs to be a fully qualified link, including armadaproject.io.  Please ensure you've reviewed https://github.com/G-Research/armada/blob/master/docs/docs-readme.md and https://github.com/G-Research/armada/blob/gh-pages/README.txt

Reply:
My apoligies, I should have checked these files. Thanks for passing that info on :))